tests/10-issue.js

```diff
@@ -26,95 +26,332 @@ describe('BitstringStatusList Credentials (Issue "statusPurpose: revocation")',
+                  const statusReferenceType = typeof (
+                    issuedVc.credentialStatus.statusReference);
+                  statusReferenceType.should.be.oneOf(['string', 'array'],
+                    'Expected statusReference to be an string or an array.');
+                  // TODO test for URLS
+                });
+              } else {
+                this.skip();
+              }
+            });
         });
-        it('MUST be 16kb in size when "encodedList" is decoded',
+        describe('BitstringStatusList Credential', function() {
+          let statusListCredential;
+          before(async function() {
+            statusListCredential = (await getSlc({issuedVc})).slc;
+          });
+          it('When a status list verifiable credential is published, ' +
+            'it MUST be a conforming document, ' +
+            'as defined in [VC-DATA-MODEL-2.0].',
+          async function() {
+            this.test.link = 'https://www.w3.org/TR/vc-bitstring-status-list/#:~:text=When%20a%20status%20list%20verifiable%20credential%20is%20published%2C%20it%20MUST%20be%20a%20conforming%20document%2C%20as%20defined%20in%20%5BVC%2DDATA%2DMODEL%2D2.0%5D';
+            testSlCredential({slCredential: statusListCredential},
+              'Expected status credential to conform to VCDM 2.0.');
+          });
+          it.skip('The verifiable credential that contains the status ' +
+            'list MAY express an id property that matches the value ' +
+            'specified in statusListCredential for the corresponding ' +
+            'BitstringStatusListEntry.',
+          async function() {
+            this.test.link = 'https://www.w3.org/TR/vc-bitstring-status-list/#:~:text=The%20verifiable%20credential%20that%20contains%20the%20status%20list%20MAY%20express%20an%20id%20property%20that%20matches%20the%20value%20specified%20in%20statusListCredential%20for%20the%20corresponding%20BitstringStatusListEntry';
+          });
+          it('The verifiable credential that contains the status list ' +
+            'MUST express a type property that includes the ' +
+            'BitstringStatusListCredential value.',
+          async function() {
+            this.test.link = 'https://www.w3.org/TR/vc-bitstring-status-list/#:~:text=The%20verifiable%20credential%20that%20contains%20the%20status%20list%20MUST%20express%20a%20type%20property%20that%20includes%20the%20BitstringStatusListCredential%20value.';
+            statusListCredential.should.have.own.property(
+              'type').to.be.an('array',
+              'Expected type property to be a string or an array.'
+            );
+            statusListCredential.type.should.include.members([
+              'BitstringStatusListCredential'
+            ],
```

Comment:
Why use `.members()` here given there's only one element? Isn't in the same as `.include('BitstringStatusListCredential')`?

Reply:
I believe I was inspired by the existing tests which do a similar syntax:
https://github.com/w3c/vc-bitstring-status-list-test-suite/blob/252ac1845950316d29eaefa5dfa6ff6ca70ddf75/tests/assertions.js#L30